openupgrade_scripts/scripts/mail/17.0.1.15/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,488 @@
+NEW ir.model.access: mail.access_discuss_channel_portal
+NEW ir.model.access: mail.access_discuss_channel_public
+NEW ir.model.access: mail.access_discuss_channel_rtc_session_system
+NEW ir.model.access: mail.access_discuss_channel_system
+NEW ir.model.access: mail.access_discuss_channel_user
+NEW ir.model.access: mail.access_discuss_gif_favorite
+NEW ir.model.access: mail.access_discuss_voice_metadata_user
+NEW ir.model.access: mail.access_mail_activity_plan_system
+NEW ir.model.access: mail.access_mail_activity_plan_template_system
+NEW ir.model.access: mail.access_mail_activity_plan_template_user
+NEW ir.model.access: mail.access_mail_activity_plan_user
+NEW ir.model.access: mail.access_mail_activity_schedule_user
+NEW ir.model.access: mail.access_mail_alias_domain_system
+NEW ir.model.access: mail.access_mail_alias_domain_user
+NEW ir.model.access: mail.access_mail_message_subtype_portal
+NEW ir.model.access: mail.access_mail_message_subtype_public
+NEW ir.model.access: mail.access_mail_message_translation_system
+NEW ir.model.access: mail.access_mail_notification_web_push
+NEW ir.model.access: mail.access_mail_partner_device
+DEL ir.model.access: mail.access_mail_activity_all
+DEL ir.model.access: mail.access_mail_activity_type_all
+DEL ir.model.access: mail.access_mail_alias_all
+DEL ir.model.access: mail.access_mail_channel_admin
+DEL ir.model.access: mail.access_mail_channel_all
+DEL ir.model.access: mail.access_mail_channel_member_portal
+DEL ir.model.access: mail.access_mail_channel_member_public
+DEL ir.model.access: mail.access_mail_channel_member_user
+DEL ir.model.access: mail.access_mail_channel_rtc_session_all
+DEL ir.model.access: mail.access_mail_channel_rtc_session_system
+DEL ir.model.access: mail.access_mail_channel_user
+DEL ir.model.access: mail.access_mail_followers_all
+DEL ir.model.access: mail.access_mail_guest_all
+DEL ir.model.access: mail.access_mail_ice_server_all
+DEL ir.model.access: mail.access_mail_mail_all
+DEL ir.model.access: mail.access_mail_mail_portal
+DEL ir.model.access: mail.access_mail_mail_user
+DEL ir.model.access: mail.access_mail_message_reaction_all
+DEL ir.model.access: mail.access_mail_message_scheduled_all
+DEL ir.model.access: mail.access_mail_message_subtype_all
+DEL ir.model.access: mail.access_mail_tracking_value_all
+DEL ir.model.access: mail.access_mail_tracking_value_portal
+DEL ir.model.access: mail.access_mail_tracking_value_user
+# NOTHING TO DO
+
+DEL ir.model.access: mail.access_res_users_settings_all [renamed to base module]
+DEL ir.model.access: mail.access_res_users_settings_user [renamed to base module]
+# NOTHING TO DO: handled in base module
+
+DEL ir.model.access: mail.access_res_users_settings_volumes_all
+DEL ir.model.constraint: mail.constraint_bus_presence_partner_or_guest_exists
+DEL ir.model.constraint: mail.constraint_mail_activity_check_res_id_is_set
+DEL ir.model.constraint: mail.constraint_mail_alias_alias_unique
```

Comment:
@remi-filament I don't think we need to remove this manually - that's necessary when we do something during migration that would violate it during migration, and afterwards it's deleted anyways